Backpack/src/main/java/net/skyscanner/backpack/chip/BpkChip.kt

```diff
@@ -18,20 +18,42 @@ open class BpkChip @JvmOverloads constructor(
   defStyleAttr: Int = 0
 ) : BpkText(wrapContextWithDefaults(context), attrs, defStyleAttr) {
 
-  init {
-    initialize(attrs, defStyleAttr)
-  }
-
   var disabled: Boolean = false
     set(value) {
       field = value
       this.isEnabled = !disabled
     }
 
+  var chipBackgroundColor: Int = -1
```

Comment:
-1 is 0xFFFFFFFF which is Color.WHITE. Use ContextCompat.getColor(context, R.color.bpkGray50) instead for the default values.